### DISCUSSION TITLE
web ui i18n not working

### DISCUSSION THREAD

**Discussion by susanforme** [verdaccio/verdaccio#4784] [General]
# env
os:windows;
config: default config;
# desc
Just now I pulled the latest image(verdaccio/verdaccio:latest) from Docker without making any modifications, just running it. Then there was an i18n error as shown in the picture. How can I solve it? thank you.
![image](https://github.com/user-attachments/assets/264550b9-b183-4a9f-bc18-d3a76db440bb)
```
main.ffdd6e67344088513fcb.js:2 language cs-CZ file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language pt-BR file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language es-ES file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language de-DE file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language it-IT file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language fr-FR file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language ja-JP file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language ru-RU file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language tr-TR file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language uk-UA file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language km-KH file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language zh-CN file not found, fallback to en-US
i @ main.ffdd6e67344088513fcb.js:2Understand this warning
main.ffdd6e67344088513fcb.js:2 language zh-TW file not found, fallback to en-US
```
this is docker logs
![image](https://github.com/user-attachments/assets/72001f2a-f9e1-41a4-954b-151cc441cf20)

**Comment by juanpicado** [comment]
Please describe every step and I will take a look thanks 🙏

**Reply by susanforme** [reply]
1. I pulled the official image(verdaccio/verdaccio:latest)
![image](https://github.com/user-attachments/assets/7cee94e9-6796-4f94-8571-2160544d7124)

2. then run
![image](https://github.com/user-attachments/assets/57de77f1-45f5-469f-8e64-d74d9367a5c5)
![image](https://github.com/user-attachments/assets/a2463213-7d9f-4bdb-ad64-4d49db13ba5f)
3. Then the problem mentioned above arises
![image](https://github.com/user-attachments/assets/3925f163-841a-4579-bd8e-1db0b529fbd8)
This is my first HTTP request to enter the page
![image](https://github.com/user-attachments/assets/258a4c9f-13c7-4bc0-a3c1-05763064d303)

**Comment by juanpicado** [comment]
🤔 works for me using `docker run -it --rm --name verdaccio -p 4873:4873 verdaccio/verdaccio:5.32.1`

**Comment by mbtools** [comment]
it works fine in 6.0.5
